app/views/foreman_virt_who_configure/api/v2/configs/main.json.rabl

```diff
@@ -4,6 +4,6 @@ extends "foreman_virt_who_configure/api/v2/configs/base"
 
 attributes :name, :interval, :organization_id, :whitelist, :blacklist, :hypervisor_id,
            :hypervisor_type, :hypervisor_server, :hypervisor_username, :debug,
-           :satellite_url, :proxy, :no_proxy, :status, :last_report_at, :out_of_date_at,
```

Comment:
Your change means that `proxy` no longer comes off the API response. Though I agree with the change conceptually and for correctness, we should keep `proxy` here API response doesn't change, I think. What do you think?

Reply:
If I'm not mistaken, there is no `proxy` attribute for configs at all.. So, even without this patch `proxy` doesn't come in the server response (checked via `hammer -d`).

BTW, @chris1984, you can try to use `http_proxy` here to retrieve `id`, `name` or `url` instead of adding a new method for the model. There is an example for children nodes somewhere...

Reply:
There used to be. When we migrated to use the HttpProxy model instead of a string, we removed it. And that's what broke the API and why we're doing this change now :)

Reply:
Went with the nested child object and output now looks like this:

```bash
[vagrant@dhcp181-248 hammer-cli-katello]$ hammer virt-who info --id 1
General information: 
    Id:                  1
    Name:                test
    Hypervisor type:     esx
    Hypervisor server:   vcenter.toledo.satellite.lab.eng.rdu2.redhat.com
    Hypervisor username: sateng
    Status:              No Report Yet
Schedule:            
    Interval:       every 2 hours
    Last Report At:
Connection:          
    Satellite server:     dhcp182-208.gsslab.rdu2.redhat.com
    Hypervisor ID:        hostname
    Filtering:            Unlimited
    Filter host parents:  
    Exclude host parents: 
    Debug mode:           no
    Ignore proxy:
HTTP Proxy:          
    HTTP proxy ID:   1
    HTTP proxy name: Food
    HTTP proxy URL:  http://ginger.lab.rdu2.redhat.com
```

Reply:
I like the hammer output!